fragments/master.sh

```diff
@@ -25,6 +25,16 @@ if ( ! ps -ef | grep "/usr/bin/docker" | grep -v 'grep' &> /dev/null ); then
     exit 1
 fi
 
+RESTART_POLICY="no"
+START_MODE=$1
+
+if [[ -n ${START_MODE} && ${START_MODE} == "install" ]]; then
```

Comment:
What do you mean exactly? Now depends on parameter we switch between restart policy so there is no need to have two different versions of start_k8s(). Maybe I have wrong understanding what is your exact idea. I need better explanation